packages/manager/apps/dedicated/client/app/account/user/users/users.html

```diff
@@ -1,7 +1,7 @@
-<div>
+<div data-ui-view>
     <div data-ovh-alert="userUsers"></div>
 
-    <oui-message data-type="info">
+    <oui-message data-ng-if="$ctrl.identityProvider == null" data-type="info">
```

Comment:
i guess you can use this:
```suggestion
    <oui-message data-ng-if="!$ctrl.identityProvider" data-type="info">
```

Reply:
Done